bio.h

```diff
@@ -6,7 +6,7 @@
 #pragma once
 
 #include "queue.h"
-#include "vhost/server.h"
+#include <vhost/server.h>
```

Comment:
Why do you think double-quotes are wrong here? These headers are a part of the project and is better to prefer headers in local include directories instead of system ones.

Reply:
>The named source file is searched for in an implementation-defined manner.
The preprocessor can not find this file in this scenation. And ""-varian does not do anythihg useful.
https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf#page=182

Reply:
I was wrong, to properly add local include directories we need `-iquote` flag instead of `-I`, and that is not supported by meson.
But this does something useful - it indicates that this header is part of the project and not an external dependency. Preprocessor will find it anyway because we add "include" directory to search paths [here](https://github.com/yandex-cloud/yc-libvhost-server/blob/master/meson.build#L7) .

> The preprocessor can not find this file in this scenation

It does find it, otherwise project wouldn't build.
If the problem you are solving is about IDE not being able to find this include, IMHO it should be solved by fixing IDE config not the code.

Reply:
> it indicates 
This is not part of the standard and not part of the general consensus. But I’m ready to remove it if I like this folk sign.
> It does find it, otherwise project wouldn't build
Of course it finds it, but not in relative paths. That's why I replaced it.